driver/src/db_interface.rs

```diff
@@ -1,200 +0,0 @@
-#[cfg(test)]
```

Comment:
I think sometimes git is not smart enough. In this situations, is better to instruct him with `git mv` so it knows that a file was moved, so it will show only the changes.